CMakeLists.txt

```diff
@@ -231,13 +231,27 @@ include(CheckCXXCompilerFlag)
 add_custom_target(
     yosys ALL 
     COMMAND $(MAKE) config-gcc
-    COMMAND $(MAKE) 
+    COMMAND make clean
+    COMMAND $(MAKE) install PREFIX=${CMAKE_CURRENT_SOURCE_DIR}/yosys/install
     WORKING_DIRECTORY ${CMAKE_CURRENT_SOURCE_DIR}/yosys
     COMMENT "Compile Yosys with given Makefile"
 )
 
 # yosys compilation ends
 
+# yosys-plugins compilation starts
+add_custom_target(
+    yosys-plugins ALL
+    COMMAND $(MAKE) clean YOSYS_PATH=${CMAKE_CURRENT_SOURCE_DIR}/yosys/install
```

Comment:
Same as a above.

Reply:
yes, this is correct, removed with https://github.com/lnis-uofu/OpenFPGA/pull/408/commits/a823c3e143ce6d67dd47d820b2eba29e498760b2